### PR TITLE
Add beta complete date and staff names

### PR DESCRIPTION
**Pull request by cathydutton**
Add date for beta completion and names for PM, DM and SM

### CODE REVIEW

File: lib/projects/pc-fish.js [hunk 1]
```diff
@@ -7,9 +7,9 @@
   "phase":"beta",
   "phase_modifier":"development",
```
Comment (liammcmurray) [inline lib/projects/pc-fish.js]: We talked about moving this to Beta Public, in which this needs to be changed from `development` to `public`

**Review by liammcmurray — CHANGES_REQUESTED** [review]
Thanks, just a question about beta status for you to check.

**Comment by cathydutton** [comment]
ok, updated to public

**Comment by liammcmurray** [comment]
LGTM 🚀

**Pull request merged**